Add doc for CalculiX adapter with PaStiX (#93)
precice/precice.github.io: pages/docs/adapters/calculix/adapter-calculix-get-adapter.md

4. Build with `make` (e.g. `make -j 4` for parallel).
5. You should now have a new executable `ccx_preCICE` in the `bin/` folder of the adapter. You may move this file to a path known by your system, or [add this to your `PATH`](https://unix.stackexchange.com/a/26059/36693) (careful when doing this!).

### Building the adapter with PaStiX

> Since version 2.17 of CalculiX, it is possible to link the PaStiX solver for increased performance, mostly through GPUs. Building the adapter with PaStix is quite tedious, as most dependencies of PaStiX and PaStiX itslef must be built from source. To help you go thorugh this, we provide [detailed instructions on a dedicated page](adapter-calculix-pastix-build.html).

Reviewer: Try using self-descriptive names of links, so that the user does not need to look at the context:

```suggestion
Since version 2.17 of CalculiX, it is possible to link the PaStiX solver for increased performance, mostly through GPUs. Building the adapter with PaStiX is quite tedious, as most dependencies of PaStiX and PaStiX itslef must be built from source. Check our [detailed instructions on building the adapter with PaStiX](adapter-calculix-pastix-build.html).
```
